tests/test_plexosdb_sqlite.py

```diff
@@ -12,8 +17,14 @@ def db_empty() -> "PlexosSQLite":
 
 
 @pytest.fixture
-def db(data_folder) -> PlexosSQLite:
-    return PlexosSQLite(xml_fname=data_folder.joinpath(DB_FILENAME))
+def db(data_folder: Path) -> Generator[PlexosSQLite, None, None]:
+    tmp_dir = TemporaryDirectory()  # dir=data_folder.parent)
+    xml_fname = data_folder / DB_FILENAME
+    xml_copy = Path(tmp_dir.name) / f"copy_{DB_FILENAME}"
+    shutil.copy(xml_fname, xml_copy)
+    db = PlexosSQLite(xml_fname=str(xml_copy))
```

Comment:
We do not need the TemporaryDirectory for this, we can just reuse the temporary file created by pytest. See: https://docs.pytest.org/en/stable/how-to/tmp_path.html

```suggestion
def db(data_folder: Path, tmp_path) -> Generator[PlexosSQLite, None, None]:
    xml_fname = tmp_path / DB_FILENAME
    xml_copy = Path(tmp_path) / f"copy_{DB_FILENAME}"
    shutil.copy(xml_fname, xml_copy)
    db = PlexosSQLite(xml_fname=str(xml_copy))
```